x/hard/keeper/liquidation.go

```diff
@@ -182,7 +182,7 @@ func (k Keeper) StartAuctions(ctx sdk.Context, borrower sdk.AccAddress, borrows,
 				}
 				// Decrement supplied coins and increment borrowed coins optimistically
 				k.DecrementSuppliedCoins(ctx, sdk.Coins{lot})
-				k.IncrementBorrowedCoins(ctx, sdk.Coins{bid})
+				k.DecrementBorrowedCoins(ctx, sdk.Coins{bid})
```

Comment:
Check error values here and update methods to avoid negative coins panic if necessary

Reply:
Yeah just saw this as well, I rushed this PR, should have looked into liquidations first

Reply:
Addresed in https://github.com/Kava-Labs/kava/pull/823/commits/9f3453a2a6723996924ca494ec838a48bfd8216f, https://github.com/Kava-Labs/kava/pull/823/commits/b67ca180d0e07403b1c76e0f4b2fc58cbeec1c85